cmd/snap/last.go

```diff
@@ -58,6 +62,11 @@ func (l *changeIDMixin) GetChangeID(cli *client.Client) (string, error) {
 	}
 
 	kind := l.LastChangeType
+	optional := false
+	if l := len(kind) - 1; kind[l] == '?' {
```

Comment:
If `kind` is empty then l will be -1. How about just using `strings.HasSuffix(kind, "?")`

Reply:
As of line 55 we know `l.Positional.ID` or `kind` are not both `""`; as of line 63 we know `kind != ""`.

Reply:
Ah, static analysis all the way. Thank you for explaining this.